apps/cli/templates/build.ts

```diff
@@ -0,0 +1,85 @@
+import { readdirSync, lstatSync, readFileSync, writeFileSync } from "node:fs";
+import { join } from "node:path";
+import { type Folder, ProjectType, ensureFileInPath } from "../src/utils";
+
+const TEMPLATES_TO_BUILD = Object.keys(ProjectType);
+
+console.log("Building Templates...");
```

Comment:
we discussed to use console.info and add a linter for .log

Reply:
This happens on the dev phase, not for the users side. We import those `.json` files in the CLI while using the template.

Reply:
so why do we need .log instead of .info?